Libraries/Geolocation/RCTLocationObserver.m

```diff
@@ -280,12 +287,9 @@ - (void)locationManager:(CLLocationManager *)manager didUpdateLocations:(NSArray
   [_pendingRequests removeAllObjects];
 
   // Stop updating if not not observing
-  if (!_observingLocation) {
+  if (!!_observingLocation) {
```

Comment:
!! is shorthand for absolute boolean.  I don't see how this is necessary if the value is "truthy".

for example:
```
console.log(!! 3);
console.log(!! "Truthy string");
console.log(!! {});
```